### PR TITLE
Added python article to convert string to integers

### PR DESCRIPTION
**Pull request by invinciblycool**
Let me know if any corrections or formatting edits are needed.
Will add more articles as soon I get tips on this article.

### CODE REVIEW

File: src/pages/python/how-to-convert-strings-into-integers-in-python/index.md [hunk 1]
```diff
@@ -3,13 +3,69 @@ title: How to Convert Strings into Integers in Python
...
+print(age_int+2)
+```
+Output:
+```py
+20
+```
+
+But you should keep in mind some special cases:
+
+1. A floating point(an integer with fractional part) as an argument will return the smallest nearest integer.
```
Comment (Bouncey) [inline src/pages/python/how-to-convert-strings-into-integers-in-python/index.md]: From the doc you linked at the end of the article

> For floating point numbers, this truncates towards zero.

So my understanding is `int(7.9) == 7.0 == 7`

> return the smallest nearest integer

Maybe `return the float rounded down to the nearest whole integer` or something along those lines would be clearer

File: src/pages/python/how-to-convert-strings-into-integers-in-python/index.md [hunk 2]
```diff
@@ -3,13 +3,69 @@ title: How to Convert Strings into Integers in Python
...
+   For example : `print(7.9)` will print `7`.
```
Comment (Bouncey) [inline src/pages/python/how-to-convert-strings-into-integers-in-python/index.md]: Am I getting confused over `print()` and `int()`?

Wouldn't `print(7.9)` just print 7.9?

Comment (invinciblycool) [reply src/pages/python/how-to-convert-strings-into-integers-in-python/index.md]: Yeah you are right, I missed that `int()`.

File: src/pages/python/how-to-convert-strings-into-integers-in-python/index.md (hunk 1 above)
Comment (invinciblycool) [reply src/pages/python/how-to-convert-strings-into-integers-in-python/index.md]: Yeah that would be better.

**Comment by invinciblycool** [comment]
Updated as requested.

**Comment by Bouncey** [comment]
Great :+1: 
## LGMT!

Thanks @invinciblycool :sparkles: :sparkles:

**Pull request merged**